types.go

```diff
@@ -22,6 +22,8 @@ type ReleaseContent struct {
 type ReleaseHistory struct {
 	// AppVersion is the app version of the Helm Chart that has been deployed.
 	AppVersion string
+	// Description is the log message about past Helm chart deployment provided by tiller API.
```

Comment:
```suggestion
	// Description is a human-friendly "log entry" about this Helm release.
```

Let's use this based on what they have in upstream.

https://github.com/helm/helm/blob/ae17f3199e519126a83409d08d534d12dadd75aa/_proto/hapi/release/info.proto#L35

Reply:
Much simple 😆